### ISSUE TITLE
Allow reissue before certificate is validated

### ISSUE THREAD

**Issue by DennisSkov** [gogetssl/whmcs-addon#171]
It would be nice if the client could reissue a certificate if something went wrong during the initial configuration of the certificate. If they made a typo in the domain name, there's no way for them to complete the verification, and the client needs to contact us so we can cancel the certificate.

**Comment by cenourinha** [comment]
I also would like to have this options. Sometimes customers insert the wrong domain and aren't able to proceed with the validation or start a new configuration.

**Comment by vlad-evg** [comment]
Yes, a known problem that forces customers to create more orders and canceling previous ones. There are technical limits on the vendors' side. Please give us a couple of days to think how we can solve that for you

**Comment by vlad-evg** [comment]
Feature scheduled on Q3 2021
